Source/MQTTnet.Server/MqttServer.cs

```diff
@@ -255,6 +255,7 @@ public Task InjectApplicationMessage(InjectedMqttApplicationMessage injectedAppl
 
         return _clientSessionsManager.DispatchApplicationMessage(
             injectedApplicationMessage.SenderClientId,
+            senderUserName: null,
```

Comment:
Consider exposing the SenderUserName as well in the injected application message. Otherwise, the new properties are null in these cases.

Reply:
OK. It's done.